src/components/EmissionsViewTable/EmissionsViewTable.js

```diff
@@ -47,10 +47,9 @@ export const EmissionsViewTable = ({ monitorPlanId, filterApply, setFilterApply
         {
             if ( viewData?.length === 0) {
                 const timeOutApply = setTimeout(() => {
-                    setMessage('Select a Reporting Period, Location, and Template and Apply Filter to view the data');
                     setPendingMessage(false)
                 }, 1000);
-                setMessage('There are no records to display');
+                setMessage('There are no records to display.\nSelect another Reporting Period, Location, and Template, then click the Apply Filter button to view the data.');
```

Comment:
You could just set the HTML here and avoid the changes below:
```suggestion
        setMessage(
          <div>
            <p>There are no records to display</p>
            <p>
              Select another Reporting Period, Location, and Template, then
              click the Apply Filter button to view the data
            </p>
          </div>
        );
```
This also avoids the extra empty `div` that is being added in the `customNoDataMessage`.